src/content/configuration/other-options.md

```diff
@@ -126,6 +126,26 @@ module.exports = {
 
 W> The final location of the cache is a combination of `cache.cacheDirectory` + `cache.name`.
 
+### `cache.cacheLocation`
+
+`string`
+
+Locations for the cache. Defaults to `cache.cacheDirectory/cache.name`.
```

Comment:
I still think this is false
```
path.resolve(cache.cacheDirectory, cache.name) !== `${cache.cacheDirectory}/${cache.name}`
```

Reply:
I guess you are right, let's go with path.resolve(cache.cacheDirectory, cache.name)